.github/workflows/ci.yml

```diff
@@ -15,7 +44,7 @@ jobs:
         uses: actions/setup-java@v3
         with:
           java-version: 17
-          distribution: 'temurin'
+          distribution: "temurin"
 
-      - name: Build with Maven
-        run: mvn verify --file skill-tree/pom.xml -Pgit-build-profile
+      - name: Integration Test
+        run: mvn verify --file skill-tree/pom.xml -Dskip-ut=true
```

Comment:
Hi @Chethan64 
If you remove the `-Pgit-build-profile` from the above command, it will not run the format check. Could you pls add it back?

Reply:
Yes, I have added it back to build steps, thanks for pointing it out